eng/Analyzers.targets

```diff
@@ -1,10 +1,13 @@
 <Project>
   <PropertyGroup>
     <!-- Disable analyzers in sourcebuild -->
-    <RunAnalyzers Condition="'$(DotNetBuildFromSource)' == 'true'">false</RunAnalyzers>
+    <RunAnalyzers Condition="'$(RunAnalyzers)' == '' And '$(DotNetBuildFromSource)' == 'true'">false</RunAnalyzers>
+    <EnableNETAnalyzers Condition="'$(EnableNETAnalyzers)' == ''">$(RunAnalyzers)</EnableNETAnalyzers>
```

Comment:
Can you remind me, what's the difference between these two properties? I guess the former is exposed in the .NET SDK and the latter is custom to dotnet/runtime? Would it make sense to merge them together in a follow-up?